ee/ucp/kubernetes/storage/use-azure-files.md

```diff
@@ -173,12 +175,12 @@ After you create a Storage Class, you can use Kubernetes
 Objects to dynamically provision Azure Files Shares. This is done using
 Kubernetes Persistent Volumes Claims
```

Comment:
Suggestion: Make "Persistent Volumes Claims" the text for the hyperlink. We don't refer to PVCs anywhere else in the document.